prettyblocks.php

```diff
@@ -126,41 +139,25 @@ public function getContent()
 
     private function loadDefault()
     {
-        Configuration::updateValue('_PRETTYBLOCKS_TOKEN_', Tools::passwdGen(25));
-
-        return true;
+        return Configuration::updateValue('_PRETTYBLOCKS_TOKEN_', Tools::passwdGen(25));
     }
 
     public function install()
     {
         return parent::install() &&
         $this->loadDefault() &&
         $this->createBlockDb() &&
-        $this->registerHook('displayHeader') &&
-        $this->registerHook('displayHome') &&
-
-        $this->registerHook('displayLeftColumn') &&
-        $this->registerHook('displayRightColumn') &&
-        $this->registerHook('displayFooter') &&
-        $this->registerHook('ActionFrontControllerSetVariables') &&
-
-        $this->registerHook('displayBlockZone') &&
-        $this->registerHook('ActionDispatcher');
+        $this->registerHook($this->hooks);
     }
 
     public function uninstall()
     {
         return parent::uninstall() &&
         $this->removeDb() &&
-        $this->unregisterHook('displayHeader') &&
-        $this->unregisterHook('displayHome') &&
-        $this->unregisterHook('displayLeftColumn') &&
-        $this->unregisterHook('displayRightColumn') &&
-        $this->unregisterHook('displayFooter') &&
-        $this->unregisterHook('ActionFrontControllerSetVariables');
+        $this->unregisterHook($this->hooks);
```

Comment:
Hi @Sinepel. I think unregister hooks is useless because already done in Module class.

Reply:
Yes, it is true, I have taken over and improved what was there before my intervention. I think it's good practice to leave them so that they are explicit.
But I can remove them.

Can you confirm @PrestaSafe @micka-fdz ?